images/sync-ghrepos-secrets-vars/Makefile

```diff
@@ -0,0 +1,23 @@
+SHELL := /bin/bash
+
+IMG_SLUG := test-infra
+IMG_NAME := pigeon
```

Comment:
We need to use the same name used in `ecr.tf` here: `sync-ghrepos-secrets-vars`!

Reply:
You're right, thanks!